ntbs-service/Services/NotificationChangesService.cs

```diff
@@ -139,10 +139,24 @@ private IEnumerable<AuditLog> GetCanonicalLogs(List<AuditLog> group)
                                                      || log.EntityType == nameof(TransferRejectedAlert));
             if (transferAlertLog != null)
```

Comment:
I think we could simplify the logic here slightly by splitting out the `TransferAlert` and `TransferRejectedAlert` cases. (My understanding is that if we view `TransferRejectedAlert` as having higher priority than `TransferAlert` by checking for it first, then there won't be any overlap between the two cases).